src/PageCharacter/CharacterDisplay/CharacterArtifactPane.tsx

```diff
@@ -57,6 +65,7 @@ function CharacterArtifactPane({ newBuild = false }: { newBuild?: boolean }) {
         <Grid container spacing={1}>
           <Grid item>
             {newBuild ? <Button onClick={equipArts} className="mr-2">Equip artifacts</Button> : <Button color="error" onClick={unequipArts}>Unequip all artifacts</Button>}
+            <Button onClick={excludeArts} sx={{ ml: 1 }}>Exclude all artifacts</Button>
```

Comment:
button should be disabled if all artifacts are all excluded. use `variant: info`

Reply:
actually, lets not do this exclusion button at all. I think this exclusion thing is something people are not using correctly. 
Once https://trello.com/c/X9og8lzc/180-priority-queue-for-use-equipped-build-option is added, this feature will be confusing to have.